phoebe/dependencies/ligeor/models/polyfit.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from operator import itemgetter
 from itertools import groupby
-from numpy.core.fromnumeric import mean
+from numpy import mean
```

Comment:
would be great to also make this suggestion upstream to https://github.com/gecheline/ligeor

Reply:
Sure, but in what form? A pull request, or just a mention somewhere?

It could also be changed to `from numpy._core.fromnumeric import mean` (see underline before `core`),  but that won't work with NumPy < 2.0.

`from numpy import mean` essentially does the same thing and works in both cases.

Reply:
Yeah, just a quick PR to that repo so the two are in sync.  Eventually we'll probably move away from packaging these within phoebe and instead have pip pull them in, so we don't want changes in phoebe that don't exist in the actual packages themselves.

Reply:
Done - [https://github.com/gecheline/ligeor/pull/1](url)